src/main/webapp/i18n/en/lectureUnit.json

```diff
@@ -149,6 +149,7 @@
                     "endPage": "End page must be less or equal to {{ max }}."
                 },
                 "split": {
+                    "empty": "",
```

Comment:
intentionally empty?

(out-of-scope here: Is a translation string really necessary, then?)

Reply:
Intentionally copied the empty string, yes :) Sound like it should not be necessary but that's way out of scope for this PR imo